src/components/ui/Shell/Shell.tsx

```diff
@@ -0,0 +1,273 @@
+  defaultVariants: {
+    intent: "primary",
+  },
+});
+
+const textCardStyles = cva("", {
+  variants: {
+    intent: {
+      primary: "text-zinc-400 dark:text-zinc-400",
+      secondary: "text-zinc-100 dark:text-zinc-100",
+    },
+  },
+  defaultVariants: {
+    intent: "primary",
+  },
+});
+
+const hrefStyles = cva("", {
+  variants: {
+    intent: {
+      primary: "text-blue-300 dark:text-blue-300",
+      secondary: "text-brand-700 dark:text-green-300",
+    },
+  },
+  defaultVariants: {
+    intent: "primary",
+  },
+});
+
+const subHeaderStyles = cva("mt-6 text-base", {
+  variants: {
+    intent: {
+      primary: "text-zinc-400 dark:text-zinc-400",
+      secondary: "text-brand-700 dark:text-green-300",
+    },
+  },
+  defaultVariants: {
+    intent: "primary",
+  },
+});
+
+const hoverStyles = cva(
+  "absolute -inset-y-6 -inset-x-4 z-0 scale-95 opacity-0 transition group-hover:scale-100 group-hover:opacity-100 sm:-inset-x-6 sm:rounded-2xl",
+  {
+    variants: {
+      intent: {
+        primary: "bg-zinc-200/50 dark:bg-zinc-800/50",
+        secondary: "bg-brand-800/50 dark:bg-green-800/50",
+      },
+    },
+    defaultVariants: {
+      intent: "primary",
+    },
+  }
+);
+
+export interface ShellProps
+  extends VariantProps<typeof shellStylesOuter>,
+    VariantProps<typeof shellStylesInner>,
+    VariantProps<typeof textStyles>,
+    VariantProps<typeof subHeaderStyles> {}
+
+interface Card {
+  title: string;
+  description: string;
+  href: string;
+}
+
+interface UISection {
+  name: string;
+  cards: Array<Card>;
+}
+
+interface UICard {
+  title: string;
+  description: string;
+  href: string;
+}
+
+export interface UISectionProps
+  extends VariantProps<typeof shellStylesOuter>,
+    VariantProps<typeof shellStylesInner>,
+    VariantProps<typeof textStyles>,
+    UISection {}
+
+export interface UIMockupCardProps
+  extends VariantProps<typeof shellStylesOuter>,
+    VariantProps<typeof shellStylesInner>,
+    VariantProps<typeof textStyles>,
+    VariantProps<typeof hrefStyles>,
+    VariantProps<typeof textCardStyles>,
+    VariantProps<typeof hoverStyles>,
+    UICard {}
+
+const sampleText =
+  "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam";
+
+const UISection = ({ ...props }: UISectionProps) => {
+  const { name, intent, cards } = props;
+  return (
+    <section className="mt-16 md:border-l md:border-zinc-100 md:pl-6 md:dark:border-zinc-700/40">
+      <div className="grid max-w-3xl grid-cols-1 items-baseline gap-y-8 md:grid-cols-4">
+        <h2
+          className={`text-sm font-semibold ${textStyles({
+            intent,
+          })}`}
+        >
+          {name}
+        </h2>
+        <div className="md:col-span-3">
+          <ul role="list" className="space-y-16">
+            {cards.map((item, index) => {
+              return (
+                <UIMockupCard
+                  key={index}
```

Comment:
In general, let's avoid using indexes as keys:
["We don’t recommend using indexes for keys if the order of items may change."](https://reactjs.org/docs/lists-and-keys.html)